checks/shell_download_validate.go

```diff
@@ -480,22 +482,28 @@ func isUnpinnedPipInstall(cmd []string) bool {
 			continue
 		}
 
-		// TODO(laurent): https://github.com/ossf/scorecard/pull/611#discussion_r660203476.
-		// Support -r <> --require-hashes.
+		// https://github.com/ossf/scorecard/issues/1306#issuecomment-974539197.
+		if strings.EqualFold(cmd[i], "--require-hashes") {
+			hashHashes = true
+			continue
```

Comment:
If at any point we find `--require-hashes`, we can set `hasUnresolvedDeps = false` and break out of this loop. 

```suggestion
			hasUnresolvedDeps = false
			break
```